website/src/content/main-concepts/user-permissions.mdx

```diff
@@ -2,10 +2,212 @@
 title: User Permissions
```

Comment:
In the index menu (as far as I understood) just three menu levels are allowed. The index menu looks a bit weird. 

@nkuehn: Am I right that there is one level too much displayed? 
In general I would vote for some additional changes:
- Level one (1.) should have the same type size like level two (2.). 
- A bigger spacing before level two is necessary to help the reader to see what belongs together.
![Index Menu](https://user-images.githubusercontent.com/58512719/73645845-66075d80-4678-11ea-9bd5-a03b5286342b.png)

Reply:
@adinakleine  let's try to keep design spacing optimizations out of content PRs - they have to happen in the docs kit repository anyways. 

"Level 1" is not a heading level but the page title which can be there only once and is derived from the name the page has in the main navigation (or the title frontmatter if the page does not have a main menu entry.)

Reply:
Thanks for the explanation. I removed all apart from one occurance of the h1.